docs/search/criteria_reference/isbookmarked_criterion.md

```diff
@@ -0,0 +1,45 @@
+---
+description: IsBookmarked Search Criterion
+---
+
+# IsBookmarked Criterion
+
+The [`IsBookmarked` Search Criterion](../../api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Repository-Values-Content-Query-Criterion-Location-IsBookmarked.html)
+searches for location based on whether it's bookmarked or not.
+
+This Criterion is available only for location Search.
+It works with current user reference.
+
+## Arguments
+
+- `value` - bool representing whether to search for bookmarked location (default `true`) or not bookmarked location (`false`)
+
+## Example
+
+### PHP
+
+``` php
+[[= include_file('code_samples/search/location/isbookmarked_criterion.php') =]]
```

Comment:
Just excluding the `<?php` file beginning so it looks like an extract of a larger code, like suggesting thatthe SearchService is injected somewhere above the sample.

```suggestion
[[= include_file('code_samples/search/location/isbookmarked_criterion.php', 2) =]]
```